parcels/examples/tutorial_parcels_structure.ipynb

```diff
@@ -0,0 +1,369 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting started with Parcels: general structure"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are many different ways in which to use Parcels for research. The flexibility of the parcels code enables this wide range of applicability and allows you to build complex simulations, but can also make your code messy which increases the chance of bugs. For a smooth programming experience with Parcels, it is recommended you make a general structure in setting up your simulations. Here we give you an overview of the main components of a Parcels simulation. These components provide you with the basic requirements for your first simulation, while the structure allows you to keep track of more complex simulations later on. A good practice is to separate the different parts of your code into sections:\n",
+    "1. [**FieldSet**](#1.-FieldSet). Load and set up the (velocity) fields that your particles need to access\n",
+    "2. [**ParticleSet**](#2.-ParticleSet). Define the type of particles you want to release, what `Variables` they have and what their initial conditions are\n",
+    "3. [**Execute kernels**](#3.-Kernel-execution). Define and compile the kernels that say what your particles need to do each timestep and execute them\n",
+    "4. [**Output**](#4.-Output). Write and store the output to a netcdf file. What you can do with the output in terms of analysis is documented [**here**](https://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/parcels/examples/tutorial_output.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: Casting lon data to np.float32\n",
+      "WARNING: Casting lat data to np.float32\n",
+      "WARNING: Casting depth data to np.float32\n",
+      "WARNING: Trying to initialize a shared grid with different chunking sizes - action prohibited. Replacing requested field_chunksize with grid's master chunksize.\n",
```

Comment:
This warning should disappear when you update to the latest master, where the dask chunking is improved?